modules/blocks/components/editable/index.js

```diff
@@ -1,3 +1,3 @@
-export default function Editable() {
-
+export default function Editable( { value } ) {
+	return wp.element.createElement( 'p', null, value );
```

Comment:
>Do you think we can add a convention of always using `el = wp.element.createElement`? Maybe not worth it here, since it's just one call

Honestly I think we should just use JSX.